src/main/java/com/uid2/admin/Constants.java

```diff
@@ -0,0 +1,7 @@
+package com.uid2.admin;
+
+public class Constants {
+    public static final String QLDB_LEDGER_NAME = "audit-logs";
```

Comment:
Can you make these a config variable name? Value will be set as environment variable.